packages/dev/docs/src/Layout.js

```diff
@@ -123,6 +123,7 @@ function Page({children, currentPage, publicUrl, styles, scripts, pathToPage}) {
   let title = currentPage.title + (!INDEX_RE.test(currentPage.name) || isBlog ? ` – ${pageSection}` : '');
   let hero = (parts.length > 1 ? HERO[parts[0]] : '') || heroImageHome;
   let heroUrl = `https://${TLD}/${currentPage.image || path.basename(hero)}`;
+  let githubLink = pathToPage.startsWith('/tmp/') ? pathToPage.replace('/tmp/', '').substring(pathToPage.indexOf('/')) : pathToPage;
```

Comment:
correct me if im wrong but i think the value of `pathToPage.indexOf('/')` will be incorrect since it won't be using the returned value of` pathToPage.replace('/tmp/', '')`. if you calculate it this way, you'll still get `/{hash}/{correct_link}`

Reply:
Ahh good catch, I had this multiline originally, and didn't catch that when condensing it. Thanks!